src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -9,13 +9,18 @@ import {
 
 import { VenueChat, PrivateChats } from "./components";
 
-import { useChatSidebarControls, useChatSidebarInfo } from "hooks/chatSidebar";
+import {
+  useChatSidebarControls,
+  useChatSidebarInfo,
+  useChatSidebarLockedOpen,
+} from "hooks/chatSidebar";
 
 import { ChatTypes } from "types/chat";
 
 import "./ChatSidebar.scss";
 
 export const ChatSidebar: React.FC = () => {
+  useChatSidebarLockedOpen();
```

Comment:
This might be a better name for it?

```suggestion
  useChatSidebarPinned();
```